src/Microsoft.Extensions.Logging.AzureAppServices/AzureAppServicesLoggerFactoryExtensions.cs

```diff
@@ -95,7 +95,7 @@ private static ConfigurationBasedLevelSwitcher CreateFileFilterConfigureOptions(
         /// Adds an Azure Web Apps diagnostics logger.
         /// </summary>
         /// <param name="factory">The extension method argument</param>
-        [Obsolete("Use AddAzureWebAppDiagnostics(this ILoggingBuilder builder) overload")]
+        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is AddAzureWebAppDiagnostics(this ILoggingBuilder builder) overload")]
```

Comment:
`alternative is THE xyz overload.` (insert `the`, add period at the end)

Reply:
Or just remove the word `overload`. So, `The recommended alternative is Foo(x, y, z).`